### ISSUE TITLE
[Question] Can we get Success/Failure status

### ISSUE THREAD

**Issue by vfdev-5** [Joel-hanson/k-action#4]
Hi @Joel-hanson 

I'm wondering if this `kaggle-kernel-actions` can be used to launch some recurrent tests as a script or a notebook and receive an output as success or failure on github CI ? 
I'm thinking to setup such actions to ignite and run some tests on TPUs. Do you think this could be possible ?  

Thanks

**Comment by Joel-hanson** [comment]
Hi @vfdev-5,

I didn't actually get your question. Are you asking If we can use this action `kaggle-kernel-actions` to run some TPU tests of Ignite on kaggle? Or generally use this action to run tests on TPU instance and receive output as success or failure.

**Comment by vfdev-5** [comment]
Well, the main question is 

> I'm wondering if this kaggle-kernel-actions can be used to launch some recurrent tests as a script or a notebook and receive an output as success or failure on github CI ?

The context of the question is

> I'm thinking to setup such actions to ignite and run some tests on TPU

**Comment by Joel-hanson** [comment]
@vfdev-5 Yes its possible if you are running the scripts to test it on kaggle kernels. Keep in mind that this action is going to have some bugs as it is in its early stage.

**Comment by vfdev-5** [comment]
@Joel-hanson would you be interested to setup such action to ignite ?

**Comment by Joel-hanson** [comment]
Yes, I would love to help you with that @vfdev-5.

**Comment by vfdev-5** [comment]
@Joel-hanson sounds great ! Can you send a draft PR for that and we could complete it with relevant information ? Thanks !

**Comment by Joel-hanson** [comment]
@vfdev-5 I am very sorry to inform you that the kaggle API does not support pushing kernels to run on TPU. I had tried it out locally before making the PR and the kernels ran only on CPU and GPU but not on TPU. Sorry for the misleading information.

**Comment by vfdev-5** [comment]
@Joel-hanson yes, seems like that. I was also wondering whether it is a limitation of python kaggle API wrapper or also of REST API too... Can you check that too ?

**Comment by Joel-hanson** [comment]
@vfdev-5 I have gone through the entire source code and didn't find any reference about TPU. I have also raised an issue regarding this (https://github.com/Kaggle/kaggle-api/issues) #286

**Comment by vfdev-5** [comment]
@Joel-hanson yes, I know that there is nothing in the repository mentioning the TPUs. My point was about exploring the original Kaggle's REST API where Kaggle Python API is just a python client. For example, when you do `kaggle kernels list`, AFAIK, it just prepares and send an HTTP request to the server. Maybe, the original REST API can have TPU options that were not ported to Python client...

**Comment by Joel-hanson** [comment]
I think there isn't one @vfdev-5, I had checked everywhere and didn't find anything related to it.
